pkg/server/combined_statement_stats.go

```diff
@@ -337,8 +321,10 @@ FROM %s %s`, table, whereClause)
 		if err != nil {
 			return 0, err
 		}
+
+		// It's possible the table is empty. Just return 0.
```

Comment:
Unrelated, but should that defer func closing the iterator be moved up to after the first error check after the query?